adapters/dmx/dmx.go

```diff
@@ -271,10 +278,15 @@ func fetchParams(params dmxExt, inst openrtb.Imp, ins openrtb.Imp, imps []openrt
 		tempimp.Secure = &intVal
 	}
 	if banner != nil {
+		if banner.H == checkValue || banner.W == checkValue {
+			banner.H = &banner.Format[0].H
```

Comment:
Banner is shared memory. You need to make a copy of it first before modifying.
```
bannerCopy := *banner
bannerCopy.H = &banner.Format[0].H
bannerCopy.W = &banner.Format[0].W
banner = &bannerCopy
```

Reply:
Forgot it was a shared in-memory